### PR TITLE
fix: prevent potential crash in handleBezier and in refreshRate due to invalid input

### PR DESCRIPTION
**Pull request by tachyglossues**
#### Describe your PR, what does it fix/add?

before, when the conf file contained anything other than a float in the 4 bezier curve generation points for annimations, it crashed Hyprland, which now returns an error message.

#### Is there anything you want to mention? (unchecked code, possible bugs, found problems, breaking compatibility, etc.)

This is my first PR, so please check that I haven't done anything wrong. 

#### Is it ready for merging, or does it need work?

It's ready

### CODE REVIEW

File: src/config/ConfigManager.cpp [hunk 1]
```diff
@@ -1821,26 +1821,20 @@ std::optional<std::string> CConfigManager::handleBezier(const std::string& comma
     const auto  ARGS = CVarList(args);
 
     std::string bezierName = ARGS[0];
+        if (ARGS[1] == "" || ARGS[2] == "" || ARGS[3] == "" || ARGS[4] == "")
```
Comment (vaxerski) [inline src/config/ConfigManager.cpp:1824]: you can check .size

Comment (tachyglossues) [reply src/config/ConfigManager.cpp:1824]:
```suggestion
        if (ARGS.size() < 5)
```

**Comment by Micovec** [comment]
Related #6249 I think that a util function should be created instead (as suggested by Vaxry in that issue).

**Comment by tachyglossues** [comment]
> Related #6249 I think that a util function should be created instead (as suggested by Vaxry in that issue).

In this PR I've just fixed #6249 and I don't know if it's better to create a function. Because a simple try/catch is enough to solve this type of problem. And from the few tests I've done I haven't found the same problem elsewhere.

**Comment by Micovec** [comment]
I don't want to be rude, but there are **many more** places with the same problem. Just look two lines up from your first change.

Also I'm not a reviewer but you don't want to do:

```cpp
...
} catch (...) {
    // An oopsie happened
}
```

It catches everything including things like [std::bad_alloc](https://en.cppreference.com/w/cpp/memory/new/bad_alloc).

**Comment by bakicelebi** [comment]
@tachyglossues are you going to continue to work on this? This issue is annoying the hell out of me with vscode autosave. If you are not gonna continue working on this I will raise my own PR.

**Comment by tachyglossues** [comment]
> @tachyglossues are you going to continue to work on this? This issue is annoying the hell out of me with vscode autosave. If you are not gonna continue working on this I will raise my own PR.

No, because the arguments are handled differently in the code, and I don't have the necessary C++ skills to refactor the code without making a disaster of it.

**Review by vignesh1507 — CHANGES_REQUESTED** [review]
You can add few type hints to your updated code snippet.

**Pull request closed without merging**